dev/Dockerfile.dev

```diff
@@ -1,4 +1,4 @@
-FROM cyberark/phusion-ruby-fips:latest
+FROM registry2.itci.conjur.net/cyberark/phusion-ruby-fips:0.11-b201fa9
```

Comment:
Should be rolled back before merge after base image release

Reply:
Will rollback before push to master

Reply:
Fixed